autoload/targets.vim

```diff
@@ -29,6 +29,20 @@ function! targets#o(trigger, typed, count)
     call s:cleanUp()
 endfunction
 
+function! s:getKeyAsStr()
+    " getchar returns an int for a regular character, and a string for a special
+    " key (such as "\<Left>").
+    let getcharOutput = getchar()
+    if type(getcharOutput) == type(0)
+        return nr2char(getcharOutput)
+    endif
+    return getcharOutput
+endfunction
+
+function! s:hasPrefix(str, prefix)
+    return a:str[:len(a:prefix)-1] ==# a:prefix
+endfunction
```

Comment:
And these two below `targets#e()`? (in this order)

(I know it's a minor thing. The idea is to (usually) introduce helper/internal functions below the function where they are first used. And if two functions are first used within the same function, introduce them below in the order in which they were called within the function above)